test/scenarios.py

```diff
@@ -123,27 +124,28 @@
                         [
                             """{
                         "peer_id": "172.20.0.5",
+                        "created_at": "*",
                         "active_states": [],
                         "cancel_states": [
                             {
-                                "by_peer": 0
+                                "by_peer": 0,
+                                "created_at": "*"
                             }
                         ],
                         "failed_states": [],
                         "transfer_type": {
                             "Incoming": [
```

Comment:
This also seems a little bit too complicated, can we do:
```json
"type": "Incoming",
"files or paths": [
{
  ...
}
]
```

Reply:
goodbye to one level of indentation :+1:

Reply:
Nice :rocket: